test/tests/transfers/multiapp_conservative_transfer/tests

```diff
@@ -4,6 +4,7 @@
   [./conservative_transfer]
     type = Exodiff
     input = master_conservative_transfer.i
+    max_parallel = 6
```

Comment:
This isn't the right fix - We should probably put the standard "errorIfParallel" stuff in this transfer until we can look at why it doesn't work for DistributedMesh. I'll accept this for now, but it's not clear to the user why you are limiting to 6 (other than you are hiding that it's actually broken).

Reply:
It is for the integration. I will fix it soon.